Month2/week1/Month2-Project Week 1-Remembering the Bases/VideoGame/Steam.cpp

```diff
@@ -0,0 +1,408 @@
+				std::cout << "you can get back to the Category Menu menu Presssing Enter or Presss 2 for go back to the main menu" << std::endl;
+				std::cin >> Choice;
+				if (!std::cin.good() || Choice >= 3 || Choice <= 1)
+				{
+					std::cin.clear();
+					std::cout << "Invalid input, Please enter your new Choice:" << std::endl;
+					std::cin >> Choice;
+				}
+				else if (std::cin.peek() == '\n' || Choice == '\n')
+				{
+					ActiveCommand = EMenuCommand::Categories;
+					return;
+				}
+				if (Choice == 2)
+				{
+					ActiveCommand = EMenuCommand::MainMenu;
+				}
+			}
+			FCategory NewCategory(Title);
+			CategoryContainer.AddCategory(NewCategory);
+			std::cout << "Category Created" << std::endl;
+			std::cout << "What do you want to do?" << std::endl;
+			std::cout << "1.Create other category" << std::endl;
+			std::cout << "2.Back to the main Menu" << std::endl;
+			std::cin >> Choice;
+			if (!std::cin.good() || Choice >= 3 || Choice <= 0) 
+			{
+				std::cin.clear();
+				std::cin.ignore(1000, '\n');
+				std::cout << "Invalid input, please enter a new Choice";
+				std::cin >> Choice;
+			}
+		}
+		else if (ChoiceAction == 2)
+		{
+			int Choice;
+			if (CategoryContainer.GetCategoryCount() == 0)
+			{
+				std::cout << "you dont have categories to edit " << std::endl;
+				std::cout << "Presss 2 for back to de Main menu" << std::endl;
+				std::cout << "Presss enter to go back to the previous menu" << std::endl;
+				std::cin >> Choice;
+				if (std::cin.peek() == '\n' || Choice == '\n')
+				{
+					ActiveCommand = EMenuCommand::Categories;
+					return;
+				}
+			}
+			else
+			{
+				for (int CategoriesIndex = 0; CategoriesIndex < 5; CategoriesIndex += 1)
+				{
+					std::cout << CategoriesIndex << " " << CategoryContainer.GetCategory(CategoriesIndex).GetCategoryTitle() << std::endl;
+				}
+				std::cout << "Please select the category that you want to eliminate:" << std::endl;
+				int SelectedToDelete;
+				std::cin >> SelectedToDelete;
+				while (!std::cin.good())
+				{
+					std::cin.clear();
+					std::cin.ignore(1000, '\n');
+					std::cout << "Invalid input, please enter  your new selection:" << std::endl;
+					std::cin >> SelectedToDelete;
+				}
+				CategoryContainer.DeleteCategory(SelectedToDelete);
+				std::cout << "Category Eliminated" << std::endl;
+				std::cout << "Presss enter to go back to the previous menu" << std::endl;
+				std::cout << "Presss 2 to back to the main menu" << std::endl;
+				std::cin >> Choice;
+				if (!std::cin.good())
+				{
+					std::cin.clear();
+					std::cin.ignore(1000, '\n');
+					std::cout << "Invalid input,please enter your New Choice :" << std::endl;
+					std::cin >> Choice;
+				}
+				if (std::cin.peek() == '\n' || Choice == '\n')
+				{
+					ActiveCommand = EMenuCommand::Categories;
+				}
+
+			}
+		}
+		else if (ChoiceAction == 3) {
+			Choice = 2;
+		}
+
+
+	} while (Choice == 1);
+	if (Choice == 2)ActiveCommand = EMenuCommand::MainMenu;
+	
+}
+
```

Comment:
There's extra whitespace here. One space should be enough + remember to use separators next time.